src/Engine/Graphics/Renderer/OpenGLRenderer.cpp

```diff
@@ -4133,8 +4133,7 @@ void OpenGLRenderer::DrawIndoorFaces() {
                         }
                     }
 
-                    // check if this face is visible through current portal node
-                    if (pCamera3D->CullFaceToFrustum(static_vertices_buff_in, &uNumVerticesa, static_vertices_calc_out, portalfrustumnorm, 4)) {
+                    if (true) {
```

Comment:
Instead of dropping the frustum culling here, could you move setting FACE_SeenByParty out of the condition? Its pretty low poly so not too bothered but would be nice to retain this optimization.

Reply:
I could not at the time because it was causing a lot of faces not to get rendered... However, I have changed the fix once again and it's now working almost perfectly like vanilla.